### PR TITLE
New stream for TV Publica

### PR DESCRIPTION
**Pull request by brunochanrio**
I found a new stream for the TV Publica channel, the stream requires a mobile user agent, i putted a user agent on it

I tested it in my Android phone with an ExoPlayer-based player and works

### CODE REVIEW

**Comment by taodicakhia** [comment]
Not playable (contain DRM key)

**Comment by Sphinxroot** [comment]
> Not playable (contain DRM key) (played by VLC)

use https://iptv.dc-team.com/query?url=https://www.youtube.com/user/TVPublicaArgentina/live

seem that https://query-streamlink.herokuapp.com/iptv-query?streaming-ip= have a problem for now

**Comment by Dum4G** [comment]
@LaneSh4d0w

**Comment by LaneSh4d0w** [comment]
Use https://query-streamlink-us.herokuapp.com/ for the time being, weird that my European one keeps failing down. Maybe due to massive requests.

File: channels/ar.m3u [hunk 1]
```diff
@@ -130,5 +130,6 @@ https://wowza.telpin.com.ar:1935/telpintv/ttv.stream/playlist.m3u8
 http://live-edge01.telecentro.net.ar/live/smil:trm.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="TVManaArgentina.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="https://i.imgur.com/GVI8kkp.jpg" group-title="Local",TV Maná Argentina (576p)
 http://streamspub.manasat.com:1935/tvar/tvmanaar2/playlist.m3u8
-#EXTINF:-1 tvg-id="TVPublica.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/0/0a/Logo_Televisi%C3%B3n_P%C3%BAblica_Argentina.png" group-title="General",TV Pública (TVP) (1080p) [Not 24/7]
-https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/user/TVPublicaArgentina/live
+#EXTINF:-1 tvg-id="TVPublica.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/0/0a/Logo_Televisi%C3%B3n_P%C3%BAblica_Argentina.png" group-title="General",TV Pública (TVP) (720p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Linux; Android 8.0.0; Pixel 2 XL Build/OPD1.170816.004) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Mobile Safari/537.36
+https://g1.vxral-hor.transport.edge-access.net/b16/ngrp:c7_vivo01_dai_source-20001_all/c7_vivo01_dai_source-20001_720p.m3u8
```
Comment (LaneSh4d0w) [inline channels/ar.m3u:135]: Almost sure the adaptive link is https://g1.vxral-hor.transport.edge-access.net/b16/ngrp:c7_vivo01_dai_source-20001_all.m3u8, to check.

Comment (LaneSh4d0w) [reply channels/ar.m3u:135]: Key retrieval gives a 404... because referrer ?

Comment (mickeymouse1234) [reply channels/ar.m3u:135]: In my country, Argentina, the stream works without problems, possibly the stream is geo-blocked

**Comment by Sphinxroot** [comment]
could be related to:
every chunk get a new key
https://vmf.edge-apps.net/keys/011A710D2A7AA3A57C55B0D8449F2520/c7_vivo01_dai_source-20001_720p-12085.key

https://vmf.edge-apps.net/  referrer

**Comment by LaneSh4d0w** [comment]
> 
> 
> could be related to: every chunk get a new key https://vmf.edge-apps.net/keys/011A710D2A7AA3A57C55B0D8449F2520/c7_vivo01_dai_source-20001_720p-12085.key
> 
> https://vmf.edge-apps.net/ referrer

Same host than in América Argentina I believe. Hope that'll work.

File: channels/ar.m3u (hunk 1 above)
Comment (freearhey) [reply channels/ar.m3u:135]: At the moment the link does not work even via VPN

**Pull request closed without merging**